staging/dex-k8s-authenticator/templates/configmap_templates.yaml

```diff
@@ -0,0 +1,13 @@
+# A ConfigMap that stores customized `dex-k8s-authenticator` HTML templates.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "dex-k8s-authenticator.fullname" . }}-html-templates
+  labels:
+    app: {{ template "dex-k8s-authenticator.fullname" . }}
+    env: {{ default "dev" .Values.global.deployEnv }}
```

Comment:
no need to have default here, values file should be enough.

Reply:
@alejandroEsc this is consistent with what we have in other templates for this package. Would you prefer to go over all templates and fix default value for `env` label?

Reply:
There are items we need to address in a lot of these charts. Let us try to correct things as we go. Please keep changes in PRs to one per chart. Thanks.